test/e2e/azure_test.go

```diff
@@ -267,6 +267,29 @@ var _ = Describe("Workload cluster creation", func() {
 		})
 	})
 
+	It("With Flatcar control-plane and worker nodes", func() {
```

Comment:
please add a Context around this It Block 

Then we should mark it as either [REQUIRED] or [OPTIONAL]. [REQUIRED] will run it make it run on every PR and require pass for merge, while [OPTIONAL] will only run it on PRs when the optional job is triggered and will run periodically on main. I think we can start with [OPTIONAL] for now, wdyt?

Reply:
Sure, `[OPTIONAL]` sounds good. I don't have experience with Ginkgo or your naming convention for tests, so I'm not sure if `Context` and `It` blocks are properly named. I'll adjust accordingly if you have some suggestions.